wiki/misc/mappings/en.md

```diff
@@ -0,0 +1,99 @@
+# Customizing the Mappings in Your Development Environment
+
+## What are mappings?
+
+Before a new Minecraft version jar is published to Mojang's servers, it goes through a process called *obfuscation*,
+where the human-readable class, field and method names are simplified to just a few letters, mainly to optimize the file
+size. In addition, obfuscation makes code very difficult to understand, because most names are replaced by a few random
+letters. This is where mappings come into play.
+
+A mapping is just a change from one name to another, in most cases an obfuscated name to a human readable one. Each
+mapping may also have extra metadata, like documentation. A set of mappings is called a "mapping set" or more often
+just "mappings". You can think of a mapping set as a translation dictionary, where each single mapping would be a
+translation of a word to another language. Here, the obfuscated names would be the language the computer uses, and
+the dictionary would help us translate that to plain english.
+
+While a simple obuscated-to-human-readable mapping set would be enough for one minecraft version, the obfuscated names
+aren't constant between different minecraft versions: `DirtBlock` could be obfuscated as `abc` in 1.19 but in 1.20 it
+could be `def`. From this arises the need to keep the obfuscation changes between versions minimal, and the problem can
+be solved with intermediate mappings, that convert those obfuscated names to names that won't change between versions,
+but still aren't readable in english. Quilt uses Hashed Mojmap, in which every class, field, and method is prefixed by
+`C_`, `f_` and `m_` respectively, followed by an 8-letter hash. While developing mods, most of the time you'll see
+Fabric's intermediary instead, which uses `class_`, `field_` and `method_`, followed by a number. Mojang also publishes
+official mappings, often called Mojmap (short for Moj(ang)-map(pings)), for every version after 1.14, but, since they don't use
+intermediate mappings, you have to do some extra processing to use them where you'd use other mapping sets. Luckily, Loom
```

Comment:
```suggestion
official mappings, often called Mojmap (short for Moj(ang)-map(pings)), for every version after 1.14. Since they don't have an
intermediate mapping, you have to do some extra processing to use them where you'd use other mapping sets. Luckily, Loom
```